src/components/imageContainer.js

```diff
@@ -0,0 +1,28 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import data from '../imageData.js';
+
+function ImageContainer() {
+  const [image] = useState(data);
```

Comment:
Do you really need a `useState` here? Because you are just using the `data` from `imageData` and don't do anything with it in terms of a `useState` hook. Otherwise you should also add the function to change the state:

```suggestion
  const [image, setImage] = useState(data);
```